src/pages/Sketch/Sketch.module.css

```diff
@@ -7,5 +7,5 @@
     align-items: center;
     position: relative;
     height: calc(100vh - 100px);
-    overflow: hidden;
+    /* overflow: hidden; */
```

Comment:
```suggestion
```
Remove it, if it's not needed anymore.

Reply:
Okay, Also the toolbox's content will continue to appear outside the canvas , Is that Okay ?
![outtoolbox (1)](https://user-images.githubusercontent.com/59613960/112496135-2a07ee00-8d8d-11eb-9b7f-438a35eb3390.png)

Reply:
> Okay, Also the toolbox's content will continue to appear outside the canvas , Is that Okay ?

@SalmaFarghaly Do you mean to say that if we change the `Canvas Color`, it won't get reflected in the toolbox, isn't it?

Reply:
No I don't mean that , I mean the "Edge mode" part and the "Undo/Redo" part that are in the toolbox appear below the horizontal light blue line that is in the photo .